### ISSUE TITLE
Conflicts with CDK V2.0

### ISSUE THREAD

**Issue by entest-hai** [aws-amplify/amplify-cli-export-construct#34]
I am currently using CDK V2.0, here is package.json. When I try to install  **npm i @aws-amplify/cdk-exported-backend@latest** I got errors 
```
{
  "name": "amplify-export-cdk",
  "version": "0.1.0",
  "bin": {
    "amplify-export-cdk": "bin/amplify-export-cdk.js"
  },
  "scripts": {
    "build": "tsc",
    "watch": "tsc -w",
    "test": "jest",
    "cdk": "cdk"
  },
  "devDependencies": {
    "@types/jest": "^26.0.10",
    "@types/node": "10.17.27",
    "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "aws-cdk": "2.18.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "aws-cdk-lib": "2.18.0",
    "constructs": "^10.0.0",
    "source-map-support": "^0.5.16"
  }
}

```

This is errors when trying to install **npm i @aws-amplify/cdk-exported-backend@latest**
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: amplify-export-cdk@0.1.0
npm ERR! Found: constructs@10.0.108
npm ERR! node_modules/constructs
npm ERR!   constructs@"^10.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer constructs@"^3.2.27" from @aws-amplify/cdk-exported-backend@0.0.5
npm ERR! node_modules/@aws-amplify/cdk-exported-backend
npm ERR!   @aws-amplify/cdk-exported-backend@"0.0.5" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by drem-darios** [comment]
I'm running into this same issue with Python. Running `pip3 install aws-amplify.cdk.exported-backend` results in the following error:
`ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
aws-cdk-lib 2.12.0 requires constructs<11.0.0,>=10.0.0, but you have constructs 3.4.5 which is incompatible.
Successfully installed aws-amplify.cdk.exported-backend-0.0.5 constructs-3.4.5`

If I try to run `cdk ls` after installing the package, I get the following error: 
`  File "/usr/local/lib/python3.9/site-packages/aws_cdk/aws_iam/__init__.py", line 638, in AddToPrincipalPolicyResult
    policy_dependable: typing.Optional[constructs.IDependable] = None,
AttributeError: module 'constructs' has no attribute 'IDependable'`

So it looks like this library is dependent on constructs 3.4.5 which conflicts with the aws-cdk-lib requirements. Any guidance on how to resolve this issue would be appreciated.

**Comment by LeeroyHannigan** [comment]
Any update on this? Currently hitting the same issue.

**Comment by chntr** [comment]
Same issue here.

**Comment by sobolk** [comment]
We shipped new beta release `0.0.6` with CDK v2 support.
